src/object/srv_obj.c

```diff
@@ -1606,6 +1621,39 @@ obj_tgt_update(struct dtx_leader_handle *dlh, void *arg, int idx,
 	return ds_obj_remote_update(dlh, arg, idx, comp_cb);
 }
 
+/* Nonnegative return codes of process_epoch */
+enum process_epoch_rc {
+	PE_OK_REMOTE,	/* OK and epoch chosen remotely */
```

Comment:
Who will use this return value? There is no use case in this patch.

Reply:
We check if the rc is PE_OK_LOCAL. If it's not LOCAL, then it's REMOTE. Having a name is better than checking if rc is 1, for instance.

Reply:
Since then, only "PE_OK_LOCAL" seems enough.